tips_n_tricks/week3/map-collisions.md

```diff
@@ -0,0 +1,151 @@
+# Map Collisions
+
+## Review
+
+So quick review on collisions, we've covered:
+
+1. distance < half widths  
+![](distance_width.gif). 
+2. standard box-box collision (AABB)
+(gif coming soon)
+3. The pixel perfect [Duck Hunt](https://www.youtube.com/watch?v=-1NyIsZXeqU)-style detection (gif coming soon)
+
+\* here's [a cart (coming soon)](#) that the above gifs came from
+
+## Map Collisions
+
+Now how about colliding with what we have drawn on the map?  
+We could so for each moving object go through each map sprite and and do box-box collision on the two, but that'd be a lot of checks!  
```

Comment:
Awkward sentence structure. "We could check each moving object against each map sprite and do..." might work better.

Reply:
lolol it's a wonder how these things get committed. Nice catch, thank you!